File: 2704-maximum-difference-by-remapping-a-digit/2704-maximum-difference-by-remapping-a-digit.java
class Solution {

    public int minMaxDifference(int num) {
       String str = Integer.toString(num);
       String t=str;
       String s1=str;
       for(int i=0;i<s1.length();i++) {
            if(s1.charAt(i)!='9') {
                s1=s1.replace(s1.charAt(i), '9');
                break;
            }
            else {
                s1=str;
            }
           
       }
       for(int i=0;i<t.length();i++) {
            if(t.charAt(i)!='0') {
            t=t.replace(t.charAt(i), '0');
                break;
            }
            else {
             t=str;
            }
           
       }
       return Integer.parseInt(s1) - Integer.parseInt(t);
       
    }
}

File: 2704-maximum-difference-by-remapping-a-digit/2704-maximum-difference-by-remapping-a-digit.py
class Solution:
    def minMaxDifference(self, num: int) -> int:
        s=str(num)
        for i in s:
            if i!='9':
                s1=s.replace(i,'9')
                break
            else:
                s1=s
        
        for i in s:
            if i!='0':
                s2=s.replace(i,'0')
                break
            else:
                s2=s
        
        return int(s1)-int(s2)
        